components/tree/TreeNode.razor.cs

```diff
@@ -337,10 +337,39 @@ public bool IsLeaf
         /// Expand the node
         /// </summary>
         /// <param name="expanded"></param>
-        public void Expand(bool expanded)
+        public async Task Expand(bool expanded)
         {
             if (Expanded == expanded) return;
             Expanded = expanded;
+
+            await TreeComponent?.OnNodeExpand(this, Expanded, new MouseEventArgs());
+        }
+
+        /// <summary>
+        /// Expand all child nodes
+        /// </summary>
+        public async Task ExpandAll()
```

Comment:
```suggestion
        internal async Task ExpandAll()
```

Reply:
if to internal
`_tree.FindFirstOrDefaultNode(n => n.Title == "C").ExpandAll();`
not work